src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/AssessmentTest.kt

```diff
@@ -2697,7 +2698,7 @@ class AssessmentTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `GET temporary accommodation assessment contains release date and accommodation required from date`() {
+  fun `GET temporary accommodation assessment contains updated release date and accommodation required from date`() {
```

Comment:
To make this test more explicit, would it be worth creating the application with `withPersonReleaseDate(originalDate)` and `withArrivalDate(originalDate)`, then creating the assessment with `withReleaseDate(updatedDate)` and `withAccommodationRequiredFromDate(updatedDate)` then asserting that the dates are equal to the `updatedDate`?